build.sbt

```diff
@@ -1,18 +1,20 @@
 import xerial.sbt.Sonatype._
 
-ThisBuild / version      := "0.0.3"
-ThisBuild / scalaVersion := "2.13.11"
+ThisBuild / version      := "0.1.0"
+ThisBuild / scalaVersion := "2.13.16"
+
+lazy val root = project.in(file(".")).settings(publish / skip := true, publishLocal / skip := true)
 
 lazy val core = crossProject(JVMPlatform, NativePlatform, JSPlatform)
   .crossType(CrossType.Pure)
-  .in(file("."))
+  .in(file("core"))
```

Comment:
I was getting weird compilation errors because of implicit root project being created, but without dependencies